velox/dwio/parquet/tests/reader/ParquetReaderTest.cpp

```diff
@@ -260,6 +260,189 @@ TEST_F(ParquetReaderTest, parseInt) {
   assertReadExpected(intSchema(), *rowReader, expected, *pool_);
 }
 
+TEST_F(ParquetReaderTest, parseUnsignedInt1) {
+  // uint.parquet holds unsigned integer columns (uint8: TINYINT, uint16:
+  // SMALLINT, uint32: INTEGER, uint64: BIGINT) and 3 rows. Data is in plain
+  // uncompressed format:
+  //   uint8: [255, 3, 3]
+  //   uint16: [65535, 2000, 3000]
+  //   uint32: [4294967295, 2000000000, 3000000000]
+  //   uint64: [18446744073709551615, 2000000000000000000, 3000000000000000000]
+  const std::string sample(getExampleFilePath("uint.parquet"));
```

Comment:
The contents for these tests are mostly duplicate code. Can you please extract common functions to reduce the code size?